### DISCUSSION TITLE
About Terminal

### DISCUSSION THREAD

**Discussion by zhaojing1987** [cockpit-project/cockpit#18884] [Q&A] [answered]
Does using cockpit.jump (/system/terminal) support passing parameters? For example, execute a default command after jumping to the page, such as' ls - l '

**Comment by marusak** [comment, marked as answer]
No, it does not. How would this work in general for any other page? Also what is your usecase? If you want to execute `ls -l` when you visit terminal page, you can just pretty much do `cockpit.script("ls -l")`
